drive/driveimpl/fileserver.go

```diff
@@ -95,11 +109,23 @@ func (s *FileServer) SetShares(shares map[string]string) {
 	}
 }
 
-// ServeHTTP implements the http.Handler interface.
+// ServeHTTP implements the http.Handler interface. In order to prevent
+// Mark-of-the-Web bypass attacks if someone visits this fileserver directly
+// within a browser, it requires that the first path element be this file
+// server's secret token. Without knowing the secret token, it's impossible
+// to construct a URL that passes validation.
```

Comment:
Let's clarify the comment a bit, since I didn't understand it the first time around. :) Maybe something like:

"...if someone visits this fileserver as http://localhost:$PORT instead of http://100.100.100.100:$PORT, ..."

Reply:
Let me know what you think of the latest comment.